services/httpbin/docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: '3.6'
+services:
+  httpbin:
+    image: ${DOCKER_REGISTRY}/httpbin:${DOCKER_IMAGE_TAG}
+    build:
+      context: ../../
+      dockerfile: services/httpbin/legacy/Dockerfile
+      labels:
+        io.simcore.key: '{"key": "simcore/services/dynamic/httpbin"}'
+        io.simcore.version: '{"version": "${DOCKER_IMAGE_TAG}"}'
+        io.simcore.type: '{"type": "dynamic"}'
+        io.simcore.name: '{"name": "httpbin"}'
+        io.simcore.description: '{"description": "Used to boot in legacy mode"}'
+        io.simcore.authors: '{"authors": [{"name": "Andrei Neagu", "email": "neagu@itis.swiss", "affiliation": "ITIS Foundation"}]}'
+        io.simcore.contact: '{"contact": "neagu@itis.swiss"}'
+        io.simcore.inputs: '{"inputs": {}}'
```

Comment:
you might want to have some inputs maybe? for a more general purpose service?
for the output I understand that might be more difficult.

Reply:
In order to go on that route, I'd actually write a service from scratch, something very simple. And we'd have complete control, like with the sleeper. 
But I understand your point.

Reply:
well I just meant that so you could probably test how inputs are brought to your service. now you have no inputs at all.
but this can be improved later.